lib/client.js

```diff
@@ -903,6 +903,37 @@ MatrixClient.prototype.getUsers = function() {
     return this.store.getUsers();
 };
 
+// User Account Data operations
+// ============================
+
+/**
+ * Set account data event for the current user.
+ * @param {string} eventType The event type
+ * @param {Object} the contents object for the event
```

Comment:
For JSDoc to pick it up sanely,  the thing after the type has to be the variable name, hence this should be:
```
* @param {Object} contents the object for the event
```